tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java

```diff
@@ -266,6 +269,26 @@ public static void openAsProject( SDK sdk )
             try
             {
                 sdkProject.create( description, npm );
+
+                IPath settingFolderPath = sdkProject.getLocation().append( ".settings" );
+                File settingFolder = settingFolderPath.toFile();
+
+                if( !settingFolder.exists() )
+                {
+                    settingFolder.mkdir();
+                }
+
+                if( !settingFolderPath.append( "org.eclipse.wst.validation.prefs" ).toFile().exists() )
+                {
+                    URL url =
+                        FileLocator.toFileURL( SDKCorePlugin.getDefault().getBundle().getEntry(
+                            "files/org.eclipse.wst.validation.prefs" ) );
+
+                    File file = new File( url.getFile() );
+
+                    FileUtil.copyFileToDir( file, settingFolder );
```

Comment:
Hey Greg, this code would check the original validation prefs file existing or not.